src/components/elements/headers.tsx

```diff
@@ -7,47 +7,47 @@ const headingLinkClasses = "[&_a]:hocus:underline";
 
 export const H1 = ({children, className, ...props}: Props) => {
   return (
-    <h1 className={twMerge(className, "text-m4 font-medium")} {...props}>
+    <h1 className={twMerge(className, "text-m4 font-normal")} {...props}>
       {children}
     </h1>
   )
 }
 
 export const H2 = ({children, className, ...props}: Props) => {
   return (
-    <h2 className={twMerge(headingLinkClasses, "text-m3 font-medium", className)} {...props}>
+    <h2 className={twMerge(headingLinkClasses, "text-m3 font-normal", className)} {...props}>
```

Comment:
From what I saw, most of the headings were `font-weight: 500`. Is the wysiwyg different?